metabase/metabase
# left align dashcard actions on small cards on the left side of the da…

Milestone 3 of https://github.com/metabase/metabase/issues/34367
### Description

Cards too small and with too many dashcard actions had the menu going outside of the viewport

### How to verify

Resize a card to 2x2 and drag it to the far left of the dashboard, actions should be left aligned


### Demo

https://github.com/metabase/metabase/assets/1914270/2bffae22-044c-4560-bf8f-469a544782d3


### Checklist
